jetty-core/pom.xml

```diff
@@ -38,6 +38,7 @@
     <module>jetty-security</module>
     <module>jetty-server</module>
     <module>jetty-session</module>
+    <module>jetty-siwe</module>
```

Comment:
Does this need to go in the `bom` as well?

Reply:
Yes, I just added to bom and also dependencyManagment of the main pom.xml.